docs/src/tut/mkt/index.md

```diff
@@ -774,8 +773,8 @@ For example, the atomic unit of `Ethereum` is `WEI`.
 The code so far has defined utilities for the DApp. 
 The next thing to do is to utilize these utilities in defining what happens if the `{!rsh} Participant` is a `seller` or a `buyer`.
```

Comment:
This is a little difficult to read. I suggest editing it for clarity. Maybe "is implement these utilities to define..."